midx.c

```diff
@@ -833,8 +841,15 @@ static int write_midx_internal(const char *object_dir, struct multi_pack_index *
 			packs.nr++;
 		}
 	}
+	
+	packs.pack_paths_checked = 0;
+	if (flags & MIDX_PROGRESS)
+		packs.progress = start_progress(_("Adding packfiles to MIDX"), 0);
```

Comment:
At one point, we were going to use ".midx" as part of the filename, but now it is just the "multi-pack-index". We still use "midx" for short in the code for shorthand. For anything user-facing, use "multi-pack-index".